src/app/lib/config/defaultVariants.js

```diff
@@ -0,0 +1,7 @@
+const defaultVariants = {
```

Comment:
Ideally this should be combined with [this logic](https://github.com/bbc/simorgh/blob/243aeb333c2397f03ec3404b6cd612eaeb8cf9a3/src/app/lib/utilities/variantHandler/index.js#L16) which has a 'default variant' (`allowedVariants[0]`)